### PR TITLE
Update streams

### PR DESCRIPTION
**Pull request by Alstruit**
Cleanup on streams.

### CODE REVIEW

File: streams/id.m3u [hunk 1]
```diff
@@ -53,7 +53,7 @@ http://livestream.biznetvideo.net/biznet_kids/smil:kids.smil/chunklist_w41736285
 #EXTINF:-1 tvg-id="BiznetLifestyle.id",Biznet Lifestyle (480p)
 http://livestream.biznetvideo.net/biznet_lifestyle/smil:lifestyle.smil/chunklist_w109806785_b2256000_slen_tkYml6a2V5bGl2ZXN0YXJ0dGltZT0wJmJpemtleWxpdmVlbmR0aW1lPTAmYml6a2V5bGl2ZUN1c3RvbVBhcmFtZXRlcj1teVBhcmFtZXRlciZiaXprZXlsaXZlaGFzaD1qZ0RLUGNsY0dkWmZ2cklEeGZNZHBUN3Zfc3JNV0NidWJudEdvOWNDWVdwY0JCUWFsTWxMUXhIc3dtNDNQYlZS.m3u8
 #EXTINF:-1 tvg-id="BNTV.id",BNTV [Geo-blocked]
-http://vod.linknetott.swiftcontent.com/Content/HLS/Live/Channel%28ch394%29/index.m3u8
+http://vod.linknetott.swiftcontent.com/Content/HLS/Live/Channel(ch394)/index.m3u8
```
Comment (Dum4G) [inline streams/id.m3u:56]: there was a commit yesterday that fixed those links for some by using encoded symbols

Comment (Alstruit) [reply streams/id.m3u]: Must be a bug with the playlist:format script on URL normalization.

File: streams/cy.m3u [hunk 2]
```diff
@@ -14,10 +10,6 @@ http://l2.cloudskep.com/ant1cm2/abr/playlist.m3u8
 https://sc-kuzeykibrissmarttv.ercdn.net/brt1hd/bant1/playlist.m3u8
 #EXTINF:-1 tvg-id="BRT2.cy",BRT 2 (720p) [Not 24/7]
 https://sc-kuzeykibrissmarttv.ercdn.net/brt2hd/bant1/playlist.m3u8
-#EXTINF:-1 tvg-id="CityChannel.cy",City Channel (720p) [Not 24/7]
-https://dev.aftermind.xyz/edge-hls/unitrust/citychannel/index.m3u8?token=8TXWzhY3h6jrzqEqu
```
Comment (BellezaEmporium) [inline streams/cy.m3u:18]: ?token=8TXWzhY3h6jrzqEqx

**Comment by BellezaEmporium** [comment]
Should we move the jp-primehome links to a specific list, it'll make the JP playlist a bit more readable.

File: streams/cy.m3u [hunk 3]
```diff
@@ -1,10 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="AdaTV.cy",ADA TV (576p)
 https://sc-kuzeykibrissmarttv.ercdn.net/adatv/bant1/playlist.m3u8
-#EXTINF:-1 tvg-id="AlfaSport.cy",Alfa Sport (1080p) [Not 24/7]
-https://dev.aftermind.xyz/edge-hls/unitrust/alfasports/index.m3u8?token=8TXWzhY3h6jrzqEqu
```
Comment (Dum4G) [inline streams/cy.m3u:5]: Make sure to fix all of the rest channels where 8TXWzhY3h6jrzqEqu -> 8TXWzhY3h6jrzqEqx

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/us_ythls.m3u [hunk 4]
```diff
@@ -1419,7 +1419,7 @@ https://ythls.armelin.one/channel/UC5V8mErVFOpcQXEb3y9IMZw.m3u8
 https://ythls.armelin.one/channel/UCH9H_b9oJtSHBovh94yB5HA.m3u8
 #EXTINF:-1 tvg-id="UNIANTV.ua",УНІАН (1080p) [Not 24/7]
 https://ythls.armelin.one/channel/UCKCVeAihEfJr-pGH7B73Wyg.m3u8
-#EXTINF:-1 tvg-id="Khabar24.kz",Хабар 24 (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="24KZ.kz",Хабар 24 (720p) [Not 24/7]
```
Comment (Dum4G) [inline streams/us_ythls.m3u:1422]:
```suggestion
#EXTINF:-1 tvg-id="24KZ.kz",24KZ (720p) [Not 24/7]
```

File: streams/kz.m3u [hunk 5]
```diff
@@ -55,9 +57,7 @@ https://tvcdn01.oktv.kz/tv/mtrk/playlist.m3u8
 http://serv25.vintera.tv:8081/novoetv/nov_tv/playlist.m3u8
 #EXTINF:-1 tvg-id="ChannelOneEurasia.kz",Первый канал Евразия (720p)
 https://1tvkz-stream.daitsuna.net/1tvkz/1tvkz/playlist.m3u8
-#EXTINF:-1 tvg-id="Khabar24.kz",Хабар 24 (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="24KZ.kz",Хабар 24 (720p) [Not 24/7]
```
Comment (Dum4G) [inline streams/kz.m3u:60]:
```suggestion
#EXTINF:-1 tvg-id="24KZ.kz",24KZ (720p) [Not 24/7]
```

File: streams/kz.m3u [hunk 6]
```diff
@@ -55,9 +57,7 @@ https://tvcdn01.oktv.kz/tv/mtrk/playlist.m3u8
...
 http://serv30.vintera.tv:8081/habar/habar24/playlist.m3u8
-#EXTINF:-1 tvg-id="Khabar24.kz",Хабар 24 (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="24KZ.kz",Хабар 24 (720p) [Not 24/7]
```
Comment (Dum4G) [inline streams/kz.m3u:62]:
```suggestion
#EXTINF:-1 tvg-id="24KZ.kz",24KZ (720p) [Not 24/7]
```

File: streams/jp.m3u [hunk 7]
```diff
@@ -1,118 +1,42 @@
...
-#EXTINF:-1 tvg-id="GolfNetwork.jp",Golf Network (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs03
 #EXTINF:-1 tvg-id="GolfNetwork.jp",Golf Network (540p)
 http://202.60.106.14:8080/1335/playlist.m3u8
-#EXTINF:-1 tvg-id="GreenChannel.jp",Green Channel (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=bs14
 #EXTINF:-1 tvg-id="GSTV.jp",GSTV (720p)
 https://gstv-tnz-gsmediastreaming.preview-jpea.channel.media.azure.net/dfd06b62-e9d1-4a7f-bcbb-89d2ecbc82ee/preview.ism/manifest(format=mpd-time-csf,audio-only=false)
 #EXTINF:-1 tvg-id="JOMLDTV.jp",Gunma TV (720p)
 https://movie.mcas.jp/switcher/smil:mcas8.smil/master.m3u8
```
Comment (Dum4G) [inline streams/jp.m3u:25]: dead

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/jp.m3u (hunk 7 above)
Comment (Alstruit) [reply streams/jp.m3u:25]: Appears to be [working](https://streamtest.in/logs/http-cdns-jp-primehome-com-8000-zhongying-live-playlist-m3u8-cid-bs14),

Comment (Dum4G) [reply streams/jp.m3u:25]: it's gunma TV that has moved

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**